templates/properties.yml

```diff
@@ -219,7 +219,7 @@ meta:
       #####################
       #bootstrap_bosh_director settings
       directors:
-      - url: https://192.168.50.3:25555
+      - url: https://192.168.50.4:25555
         username: admin
```

Comment:
What is the impact if we remove this bosh director and put one director in directors list?